uli-community/lib/uli_community_web/live/crowdsource_contributions_live.html.heex

```diff
@@ -125,35 +125,63 @@
   Crowdsource Contributions
 </.header>
 
-<.table id="crowdsource-slur-metadata-table" rows={@slurs_metadata_list}>
-  <:col :let={slur} label="Slur's Label"><%= slur.label %></:col>
-  <:col :let={slur} label="Level of Severity"><%= slur.level_of_severity || "-" %></:col>
-  <:col :let={slur} label="Is Casual?">
-    <%= if is_nil(slur.casual), do: "-", else: (slur.casual == true && "Yes") || "No" %>
-  </:col>
+<table class="min-w-full divide-y divide-gray-200 text-center">
+  <thead>
+    <tr>
+      <th class="px-4 py-3 font-semibold text-gray-500 text-sm uppercase tracking-wider">Slur's Label</th>
+      <th class="px-4 py-3 font-semibold text-gray-500 text-sm uppercase tracking-wider">Level of Severity</th>
+      <th class="px-4 py-3 font-semibold text-gray-500 text-sm uppercase tracking-wider">Is Casual?</th>
+      <th class="px-4 py-3 font-semibold text-gray-500 text-sm uppercase tracking-wider">Is Appropriated?</th>
+      <th class="px-4 py-3 font-semibold text-gray-500 text-sm uppercase tracking-wider">Appropriation Context</th>
+      <th class="px-4 py-3 font-semibold text-gray-500 text-sm uppercase tracking-wider">Meaning</th>
+      <th class="px-4 py-3 font-semibold text-gray-500 text-sm uppercase tracking-wider">Categories</th>
+    </tr>
+  </thead>
 
-  <:col :let={slur} label="Is Appropriated?">
-    <%= if is_nil(slur.appropriated), do: "-", else: (slur.appropriated == true && "Yes") || "No" %>
-  </:col>
-  <:col :let={slur} label="Appropriation Context">
-    <%= if is_nil(slur.appropriation_context),
-      do: "-",
-      else: (slur.appropriation_context == true && "Yes") || "No" %>
-  </:col>
-
-  <:col :let={slur} label="Meaning"><%= slur.meaning || "-" %></:col>
-  <:col :let={slur} label="Categories"><%= Enum.join(slur.categories || ["-"], ",") %></:col>
-  <:action :let={slur}>
-    <%= if @user_role == :admin do %>
-      <.button phx-click={
-        JS.push("open-add-slur-modal", value: %{slur: slur}) |> show_modal("add-slur-modal")
-      }>
-        Add
-      </.button>
+  <tbody class="divide-y divide-gray-200">
+    <%= for slur <- @slurs_metadata_list do %>
+      <tr class="hover:bg-orange-200">
+        <td class="px-4 py-2"><%= slur.label %></td>
+        <td class="px-4 py-2"><%= slur.level_of_severity || "-" %></td>
+        <td class="px-4 py-2">
+          <%= if is_nil(slur.casual), do: "-", else: (slur.casual == true && "Yes") || "No" %>
+        </td>
+        <td class="px-4 py-2">
+          <%= if is_nil(slur.appropriated), do: "-", else: (slur.appropriated == true && "Yes") || "No" %>
+        </td>
+        <td class="px-4 py-2">
+          <%= if is_nil(slur.appropriation_context),
+            do: "-",
+            else: (slur.appropriation_context == true && "Yes") || "No" %>
+        </td>
+        <td class="px-4 py-2"><%= slur.meaning || "-" %></td>
+        <td class="px-4 py-2">
+          <%= 
+            Enum.map(slur.categories || ["-"], fn category ->
```

Comment:
We try to avoid putting lengthy Elixir code in .heex files. If you define a funtion in `crowdsource_contributions_live.ex` file, you will be able to access and call that function in this heex file.